CODEOWNERS

```
@@ -0,0 +1,17 @@
+# This file controls automatic PR reviewer assignment. See the following docs:
+#
+# * https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
+# * https://docs.github.com/en/organizations/organizing-members-into-teams/managing-code-review-settings-for-your-team
+#
+# The goal of this file is for most PRs to automatically and fairly have one
+# maintainer set as PR reviewers. All maintainers have permission to approve 
+# and merge PRs. All PRs must be approved by at least one maintainer before being merged.
+# 
+# Where possible, prefer explicitly specifying a maintainer who is a subject
+# matter expert for a particular part of the codebase rather than using the
+# @upbound/team-extensions group.
+#
+# See also OWNERS.md for governance details
+
+# Fallback owners
+*			@upbound/extensions-maintainers
```

```suggestion
*			@ulucinar
*			@sergenyalcin
```

I haven't been able to create this team yet, so let's stick with listing individuals for now.